web-registry/package.json

```diff
@@ -29,6 +29,7 @@
     "formik-mui": "^4.0.0-alpha.3",
     "graphql": "^15.0.0",
     "graphql.macro": "^1.4.2",
+    "lodash": "^4.17.21",
```

Comment:
Added `lodash` because we were using it in the project without having it explicitly in the dependencies (was working because of peer dependencies).